canonical-websites/docs.snapcraft.io
# Remove /t/ from URL paths

Fixes #54

- Match paths without the `/t`
- In the model, give document dictionaries a "path" property, which is the link without the /t/
- Use this property for redirecting / to the home page link
- Replace links in the navigation to links without the `/t`

QA
--

`./run`, and:

- go to http://0.0.0.0:8030/ and see that you're redirected to http://0.0.0.0:8030/snap-documentation/3781 (without the `/t`)
- go to e.g. http://0.0.0.0:8030/t/getting-started/3876, and check you're redirected to http://0.0.0.0:8030/getting-started/3876 (removing the `/t`)
- check 404s still work - e.g. http://0.0.0.0:8030/nothing/9999 should show a 404 page
- check all links in the navigation don't have `/t` in them
- check other redirects work - e.g. http://0.0.0.0:8030/build-snaps/moos -> http://0.0.0.0:8030/moos-applications/7820
